import PureCloudPlatformClientV2
from PureCloudPlatformClientV2.rest import ApiException
from pprint import pprint

# Configure OAuth2 access token for authorization: PureCloud OAuth
PureCloudPlatformClientV2.configuration.access_token = 'your_access_token'
# or use get_client_credentials_token(...), get_saml2bearer_token(...), get_code_authorization_token(...) or get_pkce_token(...)

# create an instance of the API class
api_instance = PureCloudPlatformClientV2.WorkforceManagementApi();
business_unit_id = 'business_unit_id_example' # str | The ID of the business unit
activity_plan_id = 'activity_plan_id_example' # str | The ID of the activity plan to update
body = PureCloudPlatformClientV2.UpdateActivityPlanRequest() # UpdateActivityPlanRequest | body

try:
    # Update an activity plan
    api_response = api_instance.patch_workforcemanagement_businessunit_activityplan(business_unit_id, activity_plan_id, body)
    pprint(api_response)
except ApiException as e:
    print("Exception when calling WorkforceManagementApi->patch_workforcemanagement_businessunit_activityplan: %s\n" % e)